libs/server/Storage/Session/ObjectStore/SetOps.cs

```diff
@@ -686,13 +702,23 @@ public GarnetStatus SetUnionStore(byte[] key, ArgSlice[] keys, out int count)
 
                 if (status == GarnetStatus.OK)
                 {
-                    var newSetObject = new SetObject();
-                    foreach (var item in members)
+                    if (members.Count > 0)
```

Comment:
How can union result in an empty set?

Reply:
The internal SetUnion method will return an empty HashSet if all the keys don't exist